SwedbankPaySDK/Classes/SwedbankPaySDKController.swift

```diff
@@ -272,15 +296,15 @@ public final class SwedbankPaySDKController: UIViewController {
         }
     }
     
-    private func loadPage<T>(template: SwedbankPayWebContent.HTMLTemplate<T>, scriptUrl: String, delay: Bool = false, eventHandler: @escaping (T, Any?) -> Void) {
+    private func loadPage<T>(baseURL: URL?, template: SwedbankPayWebContent.HTMLTemplate<T>, scriptUrl: URL, delay: Bool = false, eventHandler: @escaping (T, Any?) -> Void) {
```

Comment:
Codacy found an issue: [Line should be 120 characters or less: currently 174 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6260585)